docs/tutorials/getting-started.md

```diff
@@ -12,7 +10,7 @@ This guide assumes the following prerequisites:
 It was tested on Ubuntu 22.04 VM (any Ubuntu from 22.04 onwards should work fine). Note that this guide might run into issues on a container (such as docker) as containers typically are lacking virtualization capabilities.
```

Comment:
- Keep this as a list (no need to mention that it was tested with a Ubuntu 22.04 VM). 
- Use admonition syntax to display the note, i.e.
 ```
 > [!NOTE]
> Add note here
```
-

Reply:
I didn't know about banner blindness until very recently. I think that sometmes it is necessary to include the admonitions, especially for critical warnings. In this case, I think it's okay to not include the note as an admonition.